docs/develop/develop-overview.md

```diff
@@ -78,7 +78,7 @@ An Office Add-in can use the Office JavaScript APIs to interact with content in
 
 #### Explore APIs with Script Lab
 
-Script Lab is an add-in that enables you to explore the Office JavaScript API and run code snippets while you're working in an Office program such as Excel or Word. It's available for free via [AppSource](https://appsource.microsoft.com/product/office/WA104380862) and is a useful tool to include in your development toolkit as you prototype and verify the functionality you want in your add-in. In Script Lab, you can access a library of built-in samples to quickly try out APIs or even use a sample as the starting point for your own code.
```

Comment:
Why remove the URL?

Reply:
We have 2 AppSource links for Script Lab now (one for WXP and the other for Outlook). It seemed like the best bet to have the reader go to the article, see the options, and determine which they want to install. I'm open to suggestions to include both links here directly, or just let me know if I'm overthinking it.

Reply:
That makes sense